rust/private/utils.bzl

```diff
@@ -241,3 +241,32 @@ def crate_name_from_attr(attr):
             "Consider adding a crate_name attribute to set a valid crate name",
         )
     return crate_name
+
+def dedent(doc_string):
+    """Tidy excess whitespace in docstrings to not break index.md
+
```

Comment:
Can you add the python docs reference here for future reference?

Reply:
done!